zio-http/src/main/scala/zio/http/Response.scala

```diff
@@ -405,7 +408,17 @@ object Response {
       Status.Ok,
     )
 
-  private lazy val contentTypeJson: Headers = Headers(Header.ContentType(MediaType.application.json).untyped)
-  private lazy val contentTypeHtml: Headers = Headers(Header.ContentType(MediaType.text.html).untyped)
-  private lazy val contentTypeText: Headers = Headers(Header.ContentType(MediaType.text.plain).untyped)
+  /**
+   * Creates a response with content-type set to text/event-stream
+   * @param data
+   *   \- stream of data to be sent as Server Sent Events
+   */
+  def sse(data: ZStream[Any, Nothing, ServerSentEvent]): Response =
```

Comment:
Maybe: `fromServerSentEvents`? (Not sure, if `sse` is commonly understood by those interested in using the protocol, then it's probably fine as `sse`.)

Reply:
even I think it is commonly used, `fromServerSentEvents` is much better.